config/locales/en-GB.yml

```diff
@@ -225,3 +225,18 @@
 
       full_messages:
         format: "%{attribute} %{message}"
+
+  mailer:
```

Comment:
Er, just `en`, I mean -- I don't think we're currently planning to offer US and UK English.